data_list.rb

```diff
@@ -0,0 +1,21 @@
+require './data_enumerable'
```

Comment:
- [ ] Please Ensure that you use `require_relative` to import DataEnumerable since using `require_relative` for files in your directory instead of `require` is a ruby convention. "`require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory."

Reply:
This change has been done well